functions/src/index.ts

```diff
@@ -28,5 +28,6 @@ import * as ReportingSync from './modules/reporting.sync.functions';
 import * as Email from './modules/email.functions';
 import * as SE from './modules/SE.functions';
 import * as Authorization from './modules/Authorization.functions';
+import * as User from './modules/Users.functions';
```

Comment:
1. Name the file `User.functions.ts`.
2. Move all the functions from `Authorization` file into `User` as they are also related to user. `Authorization` should be removed, of course.